nginx-proxy/nginx.conf

```diff
@@ -67,6 +35,7 @@ http {
       proxy_set_header        HTTP_X_FORWARDED_PROTO https;
       proxy_pass              http://prev.wellcomecollection.org;
       proxy_intercept_errors  on;
+      add_header              x-wc-invalid-location true;
```

Comment:
This has been added to see if we hit this location.
If we don't I'll remove it in another PR.